cylc/uiserver/data_store_mgr.py

```diff
@@ -229,10 +250,10 @@ def reconcile_update(self, topic, delta, w_id):
              'command': req_method,
              'req_context': w_id}
             for w_id, info in self.workflows_mgr.workflows.items())
-        gathers = ()
+        gathers: List[Any] = []
         for kwargs in req_kwargs:
             if not ids or kwargs['req_context'] in ids:
-                gathers += (workflow_request(**kwargs),)
+                gathers.append(workflow_request(**kwargs))
```

Comment:
I used a `List` type here as with `Tuple`s you have to specify each possible type within a tuple (i.e. you can have `List[Any]` meaning a list of any type, but `Tuple[Any]` actually means a tuple with one element of any type. If you have multiple values in the tuple, you must define `Tuple[str, int, bool, Any, ...]`, which is harder for these cases where we create it dynamically).

It appears to work the same way?